### ISSUE TITLE
Dot to underscore conversion in column names

### ISSUE THREAD

**Issue by murphyke** [rstats-db/RMySQL#66]
In the olden days, dots in R variable names would be translated to underscores in column names.  It seems that at some point, RMySQL stopped behaving this way.  What are the prospects of an option to restore the previous behavior?

**Comment by hadley** [comment]
Very low, because I think it's a bad idea. It's impossible to do correctly across databases because there's no predefined list of reserved words. Instead, you'll just need to quote the column names.

**Comment by murphyke** [comment]
I agree that it was not a great convention; the only argument for such an option would be the pipelines and ecosystems out there that rely on the old behavior.
